src/xdg-autostart-generator/xdg-autostart-service.c

```diff
@@ -17,6 +17,7 @@
 #include "string-util.h"
 #include "nulstr-util.h"
 #include "strv.h"
+#include "parse-util.h"
```

Comment:
Please sort the header. That's not your fault, but please also move `nulstr-util.h` to the correct place.